src/main/kotlin/com/example/ticketbookingapp/service/MovieScreeningService.kt

```diff
@@ -0,0 +1,77 @@
+﻿package com.example.ticketbookingapp.service
+
+import com.example.ticketbookingapp.domain.MovieScreening
+import com.example.ticketbookingapp.domain.MovieScreening_
+import com.example.ticketbookingapp.domain.Movie_
+import jakarta.persistence.EntityManager
+import jakarta.persistence.PersistenceContext
+import jakarta.persistence.criteria.CriteriaQuery
+import jakarta.persistence.criteria.Root
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.stereotype.Service
+import java.time.Instant
+
+@Service
+class MovieScreeningService {
+    @PersistenceContext
+    private lateinit var entityManager: EntityManager
+
+    @Transactional(rollbackFor = [Exception::class], readOnly = true)
+    fun getScreeningsInPeriod(
+        beginningOfPeriod: Instant,
```

Comment:
A co w sytuacji, gdy seans jest w różnych strefach czasowy? :)

Reply:
Wydaje mi się, że będzie działać dobrze, to znaczy jak użytkownik rest api wpisze jako 
beginningOfPeriod 2100-01-01T07:00:00.000%2B05:00
to ten endpoint otrzyma instant reprezentujący 
2100-01-01T02:00:00.000%2B00:00
(albo inaczej reprezentujący pewien Unix time)
Innymi słowy nie interesuje tego endpointu w jakiej strefie czasowej lubi reprezentować czas użytkownik.
Zadbałem również o to by w bazie danych zapisywały się Instanty w UTC wpisując w properties
spring.jpa.properties.hibernate.jdbc.time_zone=UTC